src/components/new-asset-form/patches-field.tsx

```diff
@@ -40,15 +40,14 @@ export const PatchesField = ({
     if (patchesAndAreasData.length) {
       const patches = patchesState.patches.map((patch: PropertyPatch) => {
         return (
-          <>
-            <div className="patch" key={patch.id}>
+          <React.Fragment key={patch.id}>
```

Comment:
Fixes React console "lack of unique keys" error

Reply:
Could you add the key to `div.patch`, instead of using React fragments?

Reply:
I thought it was nicer with the div but it still gives the error with the div :s But I can try something else, remove the Fragment completely and just render one div only

Reply:
Done